packages/bumbag/src/types/props.ts

```diff
@@ -4,7 +4,7 @@ export type Altitude = Flexible<'100' | '200' | '300' | '400' | '500' | '600' |
 export type BorderRadii = Flexible<'default' | '1' | '2' | '3' | '4' | '5' | '6' | '7', string>;
 export type ButtonType = 'button' | 'submit' | 'reset';
 export type Breakpoint = Flexible<'fullHD' | 'widescreen' | 'desktop' | 'tablet' | 'mobile', string>;
-export type ColumnSpread = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+export type ColumnSpread = number;
```

Comment:
Generally, I'd advise to deviate away from breaking design constraints, but I'd be happy if this were to be wrapped in a `Flexible`.

```suggestion
export type ColumnSpread = Flexible<1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12>;
```

Reply:
Oh good point, 9/10 you'll be wanting to use an integer, so having those numbers as suggestions is better.

Just updated it!